grounding_map.csv

```diff
@@ -3069,4 +3069,27 @@ ERM proteins,FPLX,ERM,,,,
 ezrin/radixin/moesin,FPLX,ERM,,,,
 ezrin-radixin-moesin,FPLX,ERM,,,,
 TEAD,FPLX,TEAD,,,,
-tea domain,FPLX,TEAD,,,,
\ No newline at end of file
+tea domain,FPLX,TEAD,,,,
+"motif interacting with Ub-containing novel DUB family",FPLX,MINDY,,,,
+"motif interacting with ubiquitin-containing novel DUB family",FPLX,MINDY,,,,
+"motif interacting with ubiquitin-containing novel deubiquinating enzyme family",FPLX,MINDY,,,,
+"MINDY deubiquitinases",FPLX,MINDY,,,,
+"josephin domain containing",FPLX,MJD,,,,
+"josephin deubiquinatases",FPLX,MJD,,,,
+"josephin deubiquinatase",FPLX,MJD,,,,
+"MJD deubiquinating enzyme",FPLX,MJD,,,,
+"MJD deubiquinating enzymes",FPLX,MJD,,,,
+"Ubiquitin C-terminal hydrolases",FPLX,UCHL,,,,
+"UCH deubiquitinating enzyme",FPLX,UCHL,,,,
+"UCH deubiquitinating enzymes",FPLX,UCHL,,,,
+"Ubiquitin C-terminal hydrolases deubiquitinating enzyme",FPLX,UCHL,,,,
+"Ubiquitin C-terminal hydrolases deubiquitinating enzymes",FPLX,UCHL,,,,
+"Ubiquitin C-terminal hydrolase family",FPLX,UCHL,,,,
+"Ubiquitin C-terminal hydrolases family",FPLX,UCHL,,,,
+"UCH family",FPLX,UCHL,,,,
+"Deubiquinating enzymes",FPLX,DUB,,,,
+"Deubiquinating enzyme",FPLX,DUB,,,,
+"DUBs",FPLX,DUB,,,,
+"BRISC deubiquitinating enzyme complex",FPLX,BRISC_complex,,,,

```

Comment:
Add `deubiquitinase` and `deubiquitinases`, and add capitalization variants for these and the "Deubiquitinating Enzyme" variations (possibly this is less necessary now than before with a different grounding pipeline, but that has been our practice).

Reply:
Done in 8647591

Reply:
Need to correct synonyms with "deubiquin", see above

Reply:
Updated synonyms in 8ebf4a6 to include variations on "deubiquin-", "deubiquitin-", and "deubiquityl-"